src/middlewares/request.middleware.js

```diff
@@ -48,9 +48,18 @@ function createAndValidateRequestBody (req, res, next) {
     path: req.body.path,
     apiVersion: apiVersions.V1,
     msgid: req.body.params.msgid,
-    result: {}
+    result: {},
+    startTime: new Date(),
+    method: req.originalMethod
   }
 
+  rspObj.telemetryData = {
```

Comment:
This data is sent to user with response ?

Reply:
No.